x/utils/key_tagger_test.go

```diff
@@ -101,7 +94,7 @@ func TestKeyTagger(t *testing.T) {
 	}
 
 	for i, tc := range cases {
-		t.Run(fmt.Sprintf("case-%d", i), func(t *testing.T) {
+		t.Run(fmt.Sprintf("case-%s", i), func(t *testing.T) {
```

Comment:
Please rename i to testName and use it as the test name. No need for the case- prefix.